src/reducers/query.js

```diff
@@ -28,6 +28,22 @@ export const queryState = {
 export default {
   query: handleActions(
     {
+      [ActionTypes.GET_SNAPSHOTS]: (state) =>
```

Comment:
I think that this is the wrong reducer.  This should be in the snapshots reducer I think.  Also, it's never getting unset on failure